dev-requirements.txt

```diff
@@ -7,3 +7,4 @@ pytest>=3.0.5
 pytest-cov==2.4.0
 six==1.10.0
 pytz
+typing==3.6.1
```

Comment:
Good point... actually `pytz` should be in `setup.py`, I'll update `setup.py` and create a proper `requirements` file for non-development.

Regarding `typing` dependency... I'm thinking in removing it since I used it just for type hinting.